### DISCUSSION TITLE
blokada reklam w aplikacjach w tv, android tv

### DISCUSSION THREAD

**Discussion by ghost** [MajkiIT/polish-ads-filter#21481] [General]
czy jest mozliwosc blokowania reklam, w yt i polsat go w android tv? bez aplikacji blokujacej

**Comment by xxcriticxx** [comment]
tylko jeśli zainstalujesz Kodi z wtyczkami

**Comment by ghost** [comment]
aha, w jaki sposob zainstalowac tam yt lub polsat go, itp?

**Comment by xxcriticxx** [comment]
https://github.com/mbebe/blomqvist or/and https://github.com/mtr81/kodi_addons

**Comment by ghost** [comment]
sprobuje, jak mi sie uda.

**Comment by xxcriticxx** [comment]
@Lukjan39 Good Luck!

**Comment by ghost** [comment]
czy tu sa intrukcje https://kodiwpigulce.pl/wtyczki/ ?

**Reply by xxcriticxx** [reply]
yes

**Comment by ghost** [comment]
sprobuje
